package.json

```diff
@@ -45,7 +45,7 @@
     "@fal-works/esbuild-plugin-global-externals": "^2.1.2",
     "gray-matter": "^4.0.3",
     "remark-frontmatter": "^4.0.1",
-    "remark-mdx-frontmatter": "^1.0.1",
+    "remark-mdx-frontmatter": "1.0.1",
```

Comment:
There appears to be a problem with `remark-mdx-frontmatter` 1.1.0 which is breaking the tests. My local machine had 1.0.1 but CI kept installing 1.1.0. This is also why #124 failed on merge.

Reply:
1.1.1 fixes this issue, removed the pinning.